libr/core/rtr.c

```diff
@@ -790,14 +790,14 @@ R_API void r_core_rtr_add(RCore *core, const char *_input) {
 	case RTR_PROTOCOL_HTTP:
 		{
 			int len;
-			char *uri = r_str_newf ("http://%s:%s/%s", host, port, file);
+			char *uri = r_str_newf ("https://%s:%s/%s", host, port, file);
 			char *str = r_socket_http_get (uri, NULL, &len);
 			if (!str) {
 				R_LOG_ERROR ("Cannot find peer");
 				r_socket_free (fd);
 				return;
 			}
-			// eprintf ("Connected to: 'http://%s:%s'\n", host, port);
+			// eprintf ("Connected to: 'https://%s:%s'\n", host, port);
```

Comment:
no, r2 doesnt do https by default, and that http server is running without ssl, so the https urls are wrong here